### DISCUSSION TITLE
AMG8833 IR 8*8 from AliExpress works

### DISCUSSION THREAD

**Discussion by tunip** [mKeRix/room-assistant#408] [Show and tell]
Would like to give feedback of an AMG8833 IR8*8 sensor (Grid-EYE) from AliExpress which works with room-assistant installed on Pi Zero W:

![2020-12-14 21-20-21-1](https://user-images.githubusercontent.com/11404472/102131287-48875080-3e52-11eb-93b4-a1dca41e26c8.png)

https://de.aliexpress.com/item/32867572849.html?spm=a2g0s.9042311.0.0.2eb94c4dLTOtAC

It is a cheaper sensor (20€) compared to other Grid-EYE ones.

**Comment by mKeRix** [comment]
Thanks for reporting this! I'll make a note to add this hint to the docs.

**Comment by Okosotthon** [comment]
Could you please share a link or a picture, that how should I wire it to the Pi Zero?

Also Am I right, that I just need to install the room assistant to the Pi, enable I2C interface, connect the cables and that is all?

**Comment by tunip** [comment]
![image](https://user-images.githubusercontent.com/11404472/107191823-0eb76e00-69ed-11eb-8018-6f77f5a513bf.png)

You connect it like every other I2C sensor on the Pi Zero (GPIO2 - SDA, GPIO3 - SCL). Please check the datasheet of your sensor if it needs 3.3V or 5V.

**Reply by stress1ner** [reply]
Hi , so the sensor can also be used to tell the temp in the room and are you guys building custom case to house the everything would like to see pics?

**Reply by tunip** [reply]
@stress1ner The temperatures from my screenshot comes from multiple Bluetooth sensors they also read by the room-assistant. The AMG8833 sensor provides a headmap and a Occupany sensor (see GridEYE sensors in the screenshot). Inside the headmap you have temps in a grid of 8x8.
For a case you need a 3D printer and Fusion 360 or FreeCAD to build your own :-). I have my Raspberry Pi Zero under my desk. The sensor is connected via a longer cable on the desk.

**Reply by mKeRix** [reply]
@stress1ner if you'd like to have a overall temperature sensor from the AMG8833 and/or D6T sensors feel free to open a feature request issue - shouldn't be too difficult to implement

**Reply by stress1ner** [reply]
ty i will , and @tunip what height and max distance do u find that the coverage of the sensor works  well at what size are the rooms and how quick does it detect presence.
